noisyatom/aboutus/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path
+from django.conf import settings
+from django.conf.urls.static import static
+from . import views
+
+app_name = 'aboutus'
+
+urlpatterns = [
+    # path('', views.index, name='index'),
+    path('team/', views.team, name='team'),
+    path('', views.aboutus, name='aboutus'),
+
+
+] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
urlpatterns is a python list "[ 1,2,3]".
But you are adding whatever the static function returns to the list with a '+'. Will this work?
What does it do and why do you need it?